src/phone-input/types.js

```diff
@@ -0,0 +1,138 @@
+  mapIsoToLabel?: mapIsoToLabelT,
+  overrides: {
+    CountrySelect?: OverrideT<*>,
+    CountrySelectDropdown?: OverrideT<*>,
+    CountrySelectDropdownListItem?: OverrideT<*>,
+    CountrySelectDropdownFlagColumn?: OverrideT<*>,
+    CountrySelectDropdownNameColumn?: OverrideT<*>,
+    CountrySelectDropdownDialcodeColumn?: OverrideT<*>,
+  },
+};
+
+export type OverridesT = {
+  Input?: OverrideT<*>,
+  CountrySelect?: OverrideT<*>,
+  CountrySelectDropdown?: OverrideT<*>,
+  CountrySelectDropdownListItem?: OverrideT<*>,
+  CountrySelectDropdownFlagColumn?: OverrideT<*>,
+  CountrySelectDropdownNameColumn?: OverrideT<*>,
+  CountrySelectDropdownDialcodeColumn?: OverrideT<*>,
+};
+
+export type PropsT = {
+  /** Sets aria-label attribute. */
+  'aria-label': ?string,
+  /** Sets aria-labelledby attribute. */
+  'aria-labelledby': ?string,
+  /** Sets aria-describedby attribute. */
+  'aria-describedby': ?string,
+  /** Current input text value. Note, this should include the dial code of the selected country. */
+  inputValue: string,
+  /** Current selected country value. Note, this expects an entire CountryT object. */
+  countryValue: CountryT,
+  /** Change handler of the text input to be called when a value is changed. */
+  onInputChange: (event: SyntheticInputEvent<HTMLInputElement>) => mixed,
+  /** Change handler of the country select to be called when a value is changed. */
+  onCountryChange: (event: CountryChangeEventT) => mixed,
+  /** A function that can be used to map iso codes to localized country names */
+  mapIsoToLabel?: mapIsoToLabelT,
+  /** Defines the size of the text input. */
+  size?: SizeT,
+  /** Defines a maximum dropdown height. The edge of the viewport will shrink the dropdown accordingly. */
+  dropdownHeight?: string,
+  /** Defines a maximum dropdown width. The edge of the viewport will shrink the dropdown accordingly. */
+  dropdownWidth?: string,
+  overrides?: OverridesT,
+};
+
+export type StatefulPhoneInputContainerPropsT = {
+  /** Sets aria-label attribute. */
+  'aria-label': ?string,
+  /** Sets aria-labelledby attribute. */
+  'aria-labelledby': ?string,
+  /** Sets aria-describedby attribute. */
+  'aria-describedby': ?string,
+  children: PropsT => React.Node,
+  initialState: StateT,
+  stateReducer: StateReducerT,
+  onInputChange: (event: SyntheticInputEvent<HTMLInputElement>) => mixed,
+  onCountryChange: (event: CountryChangeEventT) => mixed,
+  mapIsoToLabel?: mapIsoToLabelT,
+  overrides: OverridesT,
+};
+
+export type StatefulPhoneInputPropsT = {
+  /** Sets aria-label attribute. */
+  'aria-label'?: string,
+  /** Sets aria-labelledby attribute. */
+  'aria-labelledby'?: string,
+  /** Sets aria-describedby attribute. */
+  'aria-describedby'?: string,
+  initialState?: StateT,
+  stateReducer?: StateReducerT,
+  onInputChange?: (event: SyntheticInputEvent<HTMLInputElement>) => mixed,
+  onCountryChange?: (event: CountryChangeEventT) => mixed,
+  mapIsoToLabel?: mapIsoToLabelT,
```

Comment:
Could you add descriptions to all the props above? Same for the `StatefulPhoneInputContainerPropsT `.